Codemine/Extensions/CGRect+Utilities.swift

```diff
@@ -35,7 +35,7 @@ public extension CGRect {
      
      - returns: a new CGRect with the width and height reversed to those of the current one
      */
-    public func rectByReversingSize() -> CGRect {
+	public var reversingSize: CGRect {
```

Comment:
It was `reversedSize`, but then this discussion happened https://github.com/nodes-ios/Codemine/pull/34#discussion_r80390518. But I can change it back to `reversedSize` if you think it's better